### PR TITLE
Improve implementation of quantized_batch_norm

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

### CODE REVIEW

**Comment by Skylion007** [comment]
Why is trying to rebuild docker?

**Comment by cyyever** [comment]
> Why is trying to rebuild docker?

Don't know why they were triggered.

File: aten/src/ATen/native/quantized/cpu/Normalization.cpp [hunk 1]
```diff
@@ -385,19 +385,7 @@ Tensor quantized_batch_norm(
...
-
-  Tensor qy;
-  // TODO: this should arguably support 3d as well
-  qy = q_batch_norm2d_impl<false>(
-      qx,
-      weight.defined() ? std::make_optional(weight) : std::nullopt,
-      bias.defined() ? std::make_optional(bias) : std::nullopt,
-      mean, var, eps, output_scale, output_zero_point);
-  return qy;
+  return q_batch_norm_impl<false>(qx, weight_opt,bias_opt, mean, var, eps, output_scale, output_zero_point);
```
Comment (Skylion007) [inline aten/src/ATen/native/quantized/cpu/Normalization.cpp:388]: Weird spacing around bias_opt. Might trigger clang-format

**Review by albanD — APPROVED** [review]
Pretty cool!

**Pull request closed without merging**